src/components/SystemPages/NewReaderControls.vue

```diff
@@ -0,0 +1,388 @@
+              changed: newReaderPageChanged
+            }" >
+          <span :class="{ urgent: newReaderPageChanged }">
+          <button :disabled="!isValidPageSet || newReaderPageInput.length < 1" 
+          @click="setNewReader()"
+          class="rightBtn">Activate</button>
+          </span>
+        </template>
+        <p class="hint" v-if="$localData.settings.mspaMode">
+          Enter an <strong>MS Paint Adventures</strong> page number<br>
+          e.g. www.mspaintadventures.com/?s=6&p=<strong>004130</strong></p>
+        <p class="hint" v-else>
+          Enter a <strong>Homestuck.com</strong> page number between 1 and 8129.<br>
+          e.g. www.homestuck.com/story/<strong>413</strong></p>
+        <div v-if="$isNewReader">
+            <br />
+          <div class="bigButtonRow">
+            <button @click="clearNewReader()">Switch off new reader mode</button>
+          </div>
+        </div>
+        <div v-else>
+            <br />
+          <p class="hint">Or activate a preset:</p>
+          <div class="bigButtonRow">
+            <button @click="setupProblemSleuth()">Start Problem Sleuth</button>
+            <button @click="setupHomestuck()">Start Homestuck</button>
+          </div>
+        </div>
+      </div>
+
+      <!-- <pre v-if="$localData.settings.devMode">
+        newReaderPageInput: {{newReaderPageInput}}
+        newReaderPage: {{newReaderPage}}
+        $newReaderCurrent: {{$newReaderCurrent}}
+        isValidPageSet: {{isValidPageSet}}
+      </pre> -->
+
+      <div v-if="promptMspaMode" class="settings application" >
+        <dl>
+          <template v-for="boolSetting in settingListBoolean">
+            <dt :key="boolSetting.model"><label>
+              <input type="checkbox" 
+                :name="boolSetting.model" 
+                :id="boolSetting.model" 
+                v-model="$localData.settings[boolSetting.model]"
+              >{{boolSetting.label}}</label></dt> 
+              <!-- the spacing here is made of glass -->
+            <label :for="boolSetting.model">
+              <dd class="settingDesc" v-html="boolSetting.desc" />
+            </label>
+          </template>
+        </dl>
+      </div>
+    </template>
+
+    <div v-if="featureList.includes('fastforward')" class="settings application" >
+      <h3>Reading Experience</h3>
+      <dl class="fastForwardSelection">
+        <dt>
+          <input type="radio" id="fast_forward=false" :value="false" 
+            v-model="myFastForward"/>
+          <label for="fast_forward=false">Replay</label>
+        </dt>
+        <dd><label for="fast_forward=false">Read as if you were reading it live.<br>
+          All pages will be presented how they were as of the time of your most recent page. (with some minor exceptions; see 
+          <!-- Don't link to CC in setup mode (no settings page yet!) -->
+          <template v-if="false && $localData.assetDir">
```

Comment:
Perma-false - Confirm this works as intended?

Reply:
stray debug code, whoops